src/components/Alert/__stories__/Alert.stories.tsx

```diff
@@ -12,133 +16,227 @@ export default {
+                <ShowcaseItem title={caseTitle} key={index}>
+                    <Alert {...args} icon={propValue} />
+                </ShowcaseItem>
+            ))}
+        </Showcase>
+    ),
+    args: {
+        ...Default.args,
     },
-    {
-        title,
-        message,
-        theme: 'warning',
-        view: 'outlined',
-        layout: 'horizontal',
-        actions: <Alert.Action>{right}</Alert.Action>,
+};
+
+const cornersCases: Array<AlertProps['corners']> = ['rounded', 'square'];
+
+export const Corners: Story = {
+    render: (args) => (
+        <Showcase>
+            {cornersCases.map((corners, index) => (
+                <ShowcaseItem title={corners} key={index}>
+                    <Alert {...args} corners={corners} />
+                </ShowcaseItem>
+            ))}
+        </Showcase>
+    ),
+    args: {
+        ...Default.args,
+    },
+};
+
+const viewCases: Array<AlertProps['view']> = ['filled', 'outlined'];
+
+export const View: Story = {
+    render: (args) => (
+        <Showcase>
+            {viewCases.map((view, index) => (
+                <ShowcaseItem title={view} key={index}>
+                    <Alert {...args} view={view} />
+                </ShowcaseItem>
+            ))}
+        </Showcase>
+    ),
+    args: {
+        ...Default.args,
+    },
+};
+
+const layoutCases: Array<AlertProps['layout']> = ['vertical', 'horizontal'];
+
+export const Layout: Story = {
+    render: (args) => (
+        <Showcase>
+            {layoutCases.map((layout, index) => (
+                <ShowcaseItem title={layout} key={index}>
+                    <Alert {...args} layout={layout} />
+                </ShowcaseItem>
+            ))}
+        </Showcase>
+    ),
+    args: {
+        ...Default.args,
     },
+};
+
+const rightActionText = 'To the right (lose the horse)';
+const centerActionText = 'Straight (find a wife)';
+const leftActionText = 'To the left (CC 235.2)';
+
+const actionCases: Array<{
+    caseTitle: string;
+    propValue: AlertProps['actions'];
+}> = [
     {
-        message,
-        theme: 'utility',
-        view: 'outlined',
+        caseTitle: 'Full width action',
+        propValue: <Alert.Action>{rightActionText}</Alert.Action>,
     },
     {
-        title,
-        message,
-        theme: 'utility',
-        view: 'filled',
+        caseTitle: 'One action via Alert.Action component',
+        propValue: (
+            <Alert.Actions>
+                <Alert.Action>{centerActionText}</Alert.Action>
+            </Alert.Actions>
+        ),
     },
     {
-        title,
-        message,
-        icon: null,
-        theme: 'utility',
-        view: 'filled',
+        caseTitle: 'One action via array',
+        propValue: [{text: rightActionText, handler: action('actionHandler')}],
     },
     {
-        message,
-        theme: 'normal',
-        view: 'outlined',
+        caseTitle: 'Tree actions via Alert.Action component',
```

Comment:
No need to add showcases for `Alert.Action` usages. It looks the same as with an array. This could be tested via regular unit test